docs/run_test/platform.md

```diff
@@ -0,0 +1,44 @@
+# Platform
+
+- [Run on Azure](#run-on-azure)
+- [Run on Azure without deployment](#run-on-azure-without-deployment)
+- [Run on Ready computers](#run-on-ready-computers)
+
+## Run on Azure
+
+Follow the same procedure in [Getting started with Azure](quick_run.md).
+
+## Run on Azure without deployment
+In addition to deploying a new Azure server and running tests, you can skip the
```

Comment:
Install markdown extension, and try best to fix all errors. The title and content should be split by a blank line.